src/Option/README.md

```diff
@@ -0,0 +1,24 @@
+# Option:
```

Comment:
Perhaps a better name for this component would be ListBoxOption, to tether it to the ListBox more clearly.

Reply:
I like that idea 💯

Reply:
Would `Option` be something that's reusable with other components? If not, I think clarifying the name (similar to `RadioButtonGroup` / `RadioButton`) makes sense.

Reply:
I'm not sure. I don't see it as an option (lol) for other components at the moment.